recipes/visvis/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.9.3" %}
+
+package:
+  name: visvis
+  version: {{ version }}
+
+source:
+  fn: visvis-{{ version }}.zip
+  url: https://pypi.python.org/packages/91/6e/ffc2902c110de7f223ce8dff2befad266b0fe3ff4c78516fcdf1de03bd5f/visvis-{{ version }}.zip
+  md5: b63aa22318cd25b6db6cd9b47b20e3ee
+
+build:
+  script: python setup.py install
+  number: 0
+
+requirements:
+  build:
+    - python
+  run:
+    - python
+    - numpy
+    - pyopengl
+
+test:
+  imports:
+    - visvis
+
+about:
+  home: https://github.com/almarklein/visvis
+  doc_url: https://github.com/almarklein/visvis/wiki
+  dev_url: https://github.com/almarklein/visvis
+  summary: The object oriented approach to visualization
+  license: BSD 3-Clause
```

Comment:
Not a blocking change, but you should really update the `MANIFEST.in` for visvis to include the `license.txt` so that we can reference it here under the `license_file` key.